marketplace/extensions.json

```diff
@@ -1305,5 +1305,16 @@
    "kenticoVersions": ["13.0.0"],
    "category": "other",
    "tags": ["form control", "content", "web forms", "page types", "selector"]
+  },
+  {
+    "name": "Bootstrap MultiSelect Dropdown Form Control",
```

Comment:
It looks like this value is failing the build because of a validation check

```
child "name" fails because ["name" length must be less than or equal to 40 characters long]
```

Reply:
Resolved as per the validation check